modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/IgniteCacheGroupsWithRestartsTest.java

```diff
@@ -223,15 +223,15 @@ private void testFindAndDeleteGarbage(
 
     /**
      * @param ignite Ignite to execute task on.
-     * @param deleteFoundGarbage If clearing mode should be used.
+     * @param deleteFoundGarbarge If clearing mode should be used.
      * @return Result of task run.
      */
     private VisorFindAndDeleteGarbageInPersistenceTaskResult executeTask(
         IgniteEx ignite,
-        boolean deleteFoundGarbage
+        boolean deleteFoundGarbarge
```

Comment:
Is it a misprint here Garbarge ?

Reply:
https://en.oxforddictionaries.com/thesaurus/garbage